pkg/cri/sbserver/image_pull.go

```diff
@@ -130,9 +136,14 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 	defer pcancel()
 	snapshotter, err := c.snapshotterFromPodSandboxConfig(ctx, ref, r.SandboxConfig)
 	if err != nil {
+		span.RecordError(err)
 		return nil, err
 	}
 	log.G(ctx).Debugf("PullImage %q with snapshotter %s", ref, snapshotter)
+	span.SetAttributes(
+		attribute.String("imageRef", ref),
```

Comment:
Maybe just use "ref" as the name?

Reply:
I made some changes to the attribute names, based on recommednations in the https://opentelemetry.io/docs/reference/specification/common/attribute-naming/. As per that, I went with `image.ref`.